### ISSUE TITLE
X & Nostr Automated Updates

### ISSUE THREAD

**Issue by ghost** [stackernews/stacker.news#859]
**Is your feature request related to a problem? Please describe.**
Stackers often share links to tweets on SN, but the author of the original tweet almost never knows that their content is being talked about on SN (or that it's earning someone money).

**Describe the solution you'd like**
To bring attention to how much people are earning by sharing links to X & Nostr links in posts and comments, Stacker News should set up a bot that automatically replies to the original poster on X/Nostr with a link to the relevant item on SN, along with the number of sats the post or comment has stacked.

For example, if someone shared a link to my tweet on SN, the Stacker News X bot would reply to my post on X by saying something like:

"Your tweet just earned X sats on Stacker News.

Join the conversation and earn money for your tweets on SN!

https://stacker.news/items/xxxxxx"

This is a low-hanging fruit for directly connecting with people who might be able to add value on SN, and incentivizing them to join.

**Comment by ekzyis** [comment]
I suggest waiting 24-36 hours before replying on X or nostr so the sat amount can settle.

I also had a very similar idea in mind for Hacker News with [@hn](https://stacker.news/hn) but my replies got shadowbanned. Shouldn't be an issue on X though and especially not on nostr, I believe.

**Comment by kerooke** [comment]
Yes, good point, the automatic tweet should be triggered after some time period. 6 hours seems reasonable to me because the post will likely still be at it's peak of attention and it still gives the original poster some time to jump in the thread and contribute before it falls off the main page.

By 36 hours, the post is long forgotten, and almost certainly won't appear on the front page.
